examples/create_alert.yml

```diff
@@ -0,0 +1,21 @@
+---
+- hosts: localhost
+
+  tasks:
+  - name: Create an alert in ManageIQ
+    manageiq_alert:
+      miq_url: http://miq.example.com
+      miq_username: admin
+      miq_password: secret
+      miq_verify_ssl: false
+      description: Test Alert 01
+      options:
+        notifications:
+          delay_next_evaluation: 600
+          evm_event: {}
+      entity: node
+      expression:
+        eval_method: dwh_generic
```

Comment:
a lot of MIQ internal wizardry :-(
it may be useful to have 2 examples, one for "normal"alert, one for dwh alert.  in fact the later would ideally live together with the hawkular trigger definition. and IIUC it takes 2 manageiq_provider definitions for openshift+linked dwh, I'd throw those in as well...